kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/SectionEditor.vue

```diff
@@ -1,21 +1,405 @@
+      :maxlength="400"
+      :textArea="true"
+      class="description-ktextbox-style"
+    />
+
+    <hr :style="dividerStyle">
+
+    <div>
+      <h5
+        class="section-settings-heading"
+      >
+        {{ questionOrder$() }}
+      </h5>
+      <KGrid>
+        <KGridItem
+          :layout12="{ span: 6 }"
+          :layout8="{ span: 4 }"
+          :layout4="{ span: 2 }"
+        >
+          <KRadioButton
+            v-model="selectedQuestionOrder"
+            :label="randomizedLabel$()"
+            :value="true"
+            :description="randomizedOptionDescription$()"
+          />
+        </KGridItem>
+        <KGridItem
+          :layout12="{ span: 6 }"
+          :layout8="{ span: 4 }"
+          :layout4="{ span: 2 }"
+        >
+          <KRadioButton
+            v-model="selectedQuestionOrder"
+            :label="fixedLabel$()"
+            :value="false"
+            :description="fixedOptionDescription$()"
+          />
+        </KGridItem>
+      </KGrid>
+    </div>
+
+    <hr :style="dividerStyle">
+
+    <h5
+      class="section-settings-heading"
+    >
+      {{ quizResourceSelection$() }}
+    </h5>
+    <p>{{ numberOfSelectedResources$({ count: 4 }) }}</p>
+
+    <KRouterLink
+      appearance="raised-button"
+      :to="{ path: 'select-resources' }"
+      class="change-resource-button-style"
+      iconAfter="forward"
+    >
+      {{ changeResources$() }}
+    </KRouterLink>
+
+    <hr :style="dividerStyle">
+
+    <h5
+      class="section-order-style section-settings-heading"
+    >
+      {{ sectionOrder$() }}
+    </h5>
+
+
+    <DragContainer
+      :items="sectionOrderList"
+      @sort="handleSectionSort"
+    >
+      <transition-group>
+        <Draggable
+          v-for="(section,index) in sectionOrderList"
+          :key="index"
+        >
+          <DragHandle>
+            <div
+              :style="section.isActive ? activeSection : borderStyle "
+              class="section-order-list"
+            >
+              <KGrid>
+                <KGridItem
+                  :layout12="{ span: 1 }"
+                  :layout8="{ span: 1 }"
+                >
+                  <KIcon
+                    icon="dragVertical"
+                    class="space-content"
+                  />
+                </KGridItem>
+
+                <KGridItem
+                  :layout12="{ span: 6 }"
+                  :layout8="{ span: 5 }"
+                >
+                  <p class="space-content">
+                    {{ section.name.toUpperCase() }}
+                  </p>
+                </KGridItem>
+
+                <KGridItem
+                  :layout12="{ span: 5 }"
+                  :layout8="{ span: 2 }"
+                  class="current-section-style"
+                  :style="{ color: $themePalette.grey.v_700 }"
+                >
+                  <p class="current-section-text space-content">
+                    {{ section.current }}
```

Comment:
Related to my comment about handling the `@sort` event below -- since the `sectionOrderList` will be returning an array of [`QuizSection`](https://github.com/learningequality/kolibri/blob/16a9422e69a4f94782e3bcba3ba44539f84694b4/kolibri/plugins/coach/assets/src/composables/quizCreationSpecs.js#L115) objects, we'll have the `section_id` on hand and can just check to see if `quizForge.activeSection` is equal to the section iterated over here.